dockerfiles/node-bench-regression-guard/run-tests.rb

```diff
@@ -0,0 +1,111 @@
+require 'faker'
+require 'fileutils'
+require 'json'
+require 'securerandom'
+
+require 'minitest/autorun'
+
+def in_tmp_dir
+  tmpdir = "/tmp/nbrg-tests-#{SecureRandom.hex[0..10]}"
+  FileUtils.mkdir(tmpdir)
+  FileUtils.cp('node-bench-regression-guard', tmpdir)
+  Dir.chdir(tmpdir) do
+    FileUtils.mkdir('references')
+    FileUtils.mkdir('comparisons')
+    yield
+  end
+  FileUtils.remove_dir(tmpdir)
+end
+
+def generate_name
+  Faker::Dessert.flavor.gsub(/[[:space:]]/, '::').downcase
+end
+
+def generate_bench_data
+  base = rand(100000000...200000000)
+  name = "#{generate_name}::#{generate_name}"
```

Comment:
I know it's unlikely, but we could end up with name collisions here if luck's not on our side